src/main/java/frc/robot/RobotContainer.java

```diff
@@ -7,51 +7,90 @@
+import frc.robot.subsystems.DriveBase;
+import frc.robot.subsystems.Harvester;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import edu.wpi.first.wpilibj.VictorSP;
+import edu.wpi.first.wpilibj.Solenoid;
 
 /**
- * This class is where the bulk of the robot should be declared.  Since Command-based is a
- * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
- * periodic methods (other than the scheduler calls).  Instead, the structure of the robot
- * (including subsystems, commands, and button mappings) should be declared here.
+ * This class is where the bulk of the robot should be declared. Since
+ * Command-based is a "declarative" paradigm, very little robot logic should
+ * actually be handled in the {@link Robot} periodic methods (other than the
+ * scheduler calls). Instead, the structure of the robot (including subsystems,
+ * commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
 
-  private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
+  public static final DriveBase driveBase = new DriveBase(Constants.LEFT_DRIVE_MOTOR_PORT, Constants.RIGHT_DRIVE_MOTOR_PORT,
+      Constants.LEFT_DRIVE_ENCODER_PORT_A, Constants.LEFT_DRIVE_ENCODER_PORT_B, Constants.RIGHT_DRIVE_ENCODER_PORT_A,
+      Constants.RIGHT_DRIVE_ENCODER_PORT_B, Constants.SHIFTER_PORT);
+
+
+
+  private static final VictorSP intakeMotor = new VictorSP(Constants.INTAKE_MOTOR_PORT);
+  private static final Solenoid deployPiston = new Solenoid(Constants.DEPLOY_PISTON_PORT);
 
+  public static final Harvester harvester = new Harvester(intakeMotor, deployPiston);
 
+  
+  EnhancedJoystick leftJoystick = new EnhancedJoystick(Constants.LEFT_JOYSTICK_PORT);
+  EnhancedJoystick rightJoystick = new EnhancedJoystick(Constants.RIGHT_JOYSTICK_PORT);
+  XboxController manipulator = new XboxController(Constants.MANIPULATOR_PORT);
+  
+  public static final DriveBase driveBase = new DriveBase(new WPI_TalonSRX(Constants.LEFT_DRIVE_MOTOR_0_DEVICE_NUMBER),
+  new WPI_VictorSPX(Constants.LEFT_DRIVE_MOTOR_1_DEVICE_NUMBER),
+  new WPI_VictorSPX(Constants.LEFT_DRIVE_MOTOR_2_DEVICE_NUMBER),
+  new WPI_TalonSRX(Constants.RIGHT_DRIVE_MOTOR_0_DEVICE_NUMBER),
+  new WPI_VictorSPX(Constants.RIGHT_DRIVE_MOTOR_1_DEVICE_NUMBER),
+  new WPI_VictorSPX(Constants.RIGHT_DRIVE_MOTOR_2_DEVICE_NUMBER), Constants.LEFT_SHIFTER_PORT,
+  Constants.RIGHT_SHIFTER_PORT, Constants.LEFT_DRIVE_ENCODER_PORT_A, Constants.LEFT_DRIVE_ENCODER_PORT_B,
+  Constants.RIGHT_DRIVE_ENCODER_PORT_A, Constants.RIGHT_DRIVE_ENCODER_PORT_B);
+
+  public static final Harvester harvester = new Harvester(Constants.INTAKE_MOTOR_PORT, Constants.DEPLOY_PISTON_PORT);
```

Comment:
Remember to initialize and pass your motor controller objects here. Harvester motor can be inverted here too.